### PR TITLE
Do not output HTML comments

### PR DESCRIPTION
**Pull request by R4356th**
Are they really necessary? They are currently adding performance overhead.

### CODE REVIEW

**Review by RhinosF1 — APPROVED** [review]
I'm 50/50 on how much it's needed but the code is fine.

File: includes/MatomoAnalyticsHooks.php [hunk 1]
```diff
@@ -67,8 +67,6 @@ public static function matomoScript( $skin, &$text = '' ) {
 			$cookieDisable = (int)$config->get( 'MatomoAnalyticsDisableCookie' );
 			$forceGetRequest = (int)$config->get( 'MatomoAnalyticsForceGetRequest' );
 			$text .= <<<SCRIPT
-				<!-- Matomo -->
-				<script type="text/javascript">
```
Comment (Universal-Omega) [inline includes/MatomoAnalyticsHooks.php:71]: Why remove the `<script type="text/javascript">`?

**Comment by Universal-Omega** [comment]
I don't really think this needs done to be honest. If this does cause performance issues (which I don't personally think it would) it would be extremely minor ones.

File: includes/MatomoAnalyticsHooks.php (hunk 1 above)
Comment (R4356th) [reply includes/MatomoAnalyticsHooks.php:71]: By mistake! Fixed.

Comment (Universal-Omega) [reply includes/MatomoAnalyticsHooks.php:71]: Thanks!

**Comment by R4356th** [comment]
@RhinosF1 and @Universal-Omega, even removing a single bit of unnecessary code is an improvement. Quoting Ladsgroup from [this upstream task for Vector](https://phabricator.wikimedia.org/T281025), "We are enjoying an effortless and easy and cheap internet connection. Not everyone in the world is so lucky." (Not everyone includes me, too.)

**Comment by R4356th** [comment]
Anyway, should I do a version bump and update the CHANGELOG now that I have amended this to remove unnecessary ""type="text/javascript""?

**Comment by R4356th** [comment]
> 
> 
> Anyway, should I do a version bump and update the CHANGELOG now that I have amended this to remove unnecessary ""type="text/javascript""?

Just BOLDly did that.

**Comment by R4356th** [comment]
Bump. :)

**Comment by dmehus** [comment]
I //do// agree this is quite a minor commit, but on the whole, I don't see any reason //not// to merge it, since it has no impacts on end users. If anything, it will have a micro positive impact in terms of marginal reduced page load times, I suppose. :+1:

File: extension.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
 		"Southparkfan"
 	],
 	"url": "https://github.com/miraheze/MatomoAnalytics",
-	"version": "1.0.5.9",
+	"version": "1.0.6.0",
```
Comment (Universal-Omega) [inline extension.json:8]: 1.0.5.10

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: extension.json (hunk 2 above)
Comment (R4356th) [reply extension.json]: Done.

File: CHANGELOG [hunk 3]
```diff
@@ -1,4 +1,7 @@
 == ChangeLog for MatomoAnalytics ==
+=== 1.0.5.10 (11-05-2021) ===
```
Comment (Universal-Omega) [inline CHANGELOG:2]: Last thing, then I'll merge: change the date to today.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**